### ISSUE TITLE
win_updates expansion 

### ISSUE THREAD

**Issue by michelleperz** [ansible/ansible-modules-extras#921]
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

win_updates module
##### ANSIBLE VERSION

N/A
##### SUMMARY

To mitigate the risk of pushing out security updates to prod that may not have been tested on dev or staging, it would be nice to have an additional option in this module wherein updates could be applied based on KB#.

**Comment by trondhindenes** [comment]
That's a great idea. The underlying module (PsWindowsUpdate) supports installing a specific KB thru the "-UpdateID" param. If we surfaced that as an Ansible param/option would that solve your problem?

**Comment by ansibot** [comment]
@mattdavispdx, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@mattdavispdx, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by jhawkesworth** [comment]
hmm, I imagine bot should be poking @nitzmahone  (perhaps just misconfigured).

bot_broken

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by dagwieers** [comment]
@michelleperz Could we please move this issue over to https://github.com/ansible/ansible if this is still considered valuable ? Otherwise it may not get the proper attention it may need.

Thanks in advance !

**Comment by dagwieers** [comment]
Seems a useful request still.
This issue was moved to ansible/ansible#25282

**Comment by dagwieers** [comment]
@gundalow Please close this issue.

**Comment by jhawkesworth** [comment]
closing superceeded by  ansible/ansible#25282
